static/schemas/obms/amt-obm-service.json

```diff
@@ -0,0 +1,29 @@
+{
+    "title": "amt-obm-service",
+    "definitions": {
```

Comment:
@dalebremner : this is the definition of `definitions` in the JSON-Schema: http://json-schema.org/latest/json-schema-validation.html#anchor94:
```
This keyword plays no role in validation per se. Its role is to provide a standardized location for schema authors to inline JSON Schemas into a more general schema.
```
If you want it in validation, then it's better not place it in `definitions`, if you just want to provide a partial of a larger schema, place it in `definition` is reasonable.  So just to confirm the usage?